src/clzones.cpp

```diff
@@ -288,35 +291,45 @@ bool zone_manager::zone_data::set_name()
     return false;
 }
 
-bool zone_manager::zone_data::set_type()
+bool zone_data::set_type()
 {
-    const auto maybe_type = get_manager().query_type();
+    const auto maybe_type = zone_manager::get_manager().query_type();
     if( maybe_type.has_value() && maybe_type.value() != type ) {
         auto new_options = zone_options::create( maybe_type.value() );
         if( new_options->query_at_creation() ) {
             type = maybe_type.value();
             options = new_options;
-            get_manager().cache_data();
+            zone_manager::get_manager().cache_data();
             return true;
         }
     }
     return false;
 }
 
-void zone_manager::zone_data::set_position( const std::pair<tripoint, tripoint> position )
+void zone_data::set_position( const std::pair<tripoint, tripoint> position,
+                              const bool manual )
 {
+    if( is_vehicle && manual ) {
+        popup( "You cannot move a loot zone tied to a vehicle.", PF_NONE );
```

Comment:
This probably needs translation. Or make it a `debugmsg` if it only happens as a bug.

Reply:
Because it's occurring while editing Zones, the standard message window isn't being rendered. How do I apply/enable translation effects for a popup?